client/my-sites/site-settings/form-general.jsx

```diff
@@ -815,6 +861,7 @@ const connectComponent = connect( ( state ) => {
 		siteIsJetpack: isJetpackSite( state, siteId ),
 		siteSlug: getSelectedSiteSlug( state ),
 		hasSubscriptionGifting: siteHasFeature( state, siteId, WPCOM_FEATURES_SUBSCRIPTION_GIFTING ),
+		hasSitePreviewLink: isBusinessPlan( productSlug ) || isEcommercePlan( productSlug ),
```

Comment:
Can we change those to check if the site has access to a particular feature using `siteHasFeature`?

Reply:
Yes!, it's a much better approach. 
Done it here https://github.com/Automattic/wp-calypso/pull/70242/commits/099c6fd19a058d95f7de151e781bc94cbe8b9245
Used the WPCOM feature introduced yesterday in D93335-code